app/main.py

```diff
@@ -1,6 +1,19 @@
+import functools
 from typing import Callable
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    functools.wraps
```

Comment:
what it is?

read what it is for and how to use it 
https://www.geeksforgeeks.org/python-functools-wraps-function/
https://docs.python.org/uk/3/library/functools.html
https://stackoverflow.com/questions/308999/what-does-functools-wraps-do